src/exasol/vscl/queryrenderer/AbstractQueryAppender.lua

```diff
@@ -142,4 +152,18 @@ function AbstractQueryAppender:_append_string_literal(literal)
     self:_append("'")
 end
 
+local DEFAULT_IDENTIFIER_QUOTE<const> = '"'
```

Comment:
Move this up and use it in the other constant to avoid coupling.

Reply:
Good point!